### PR TITLE
Url updates

### PR DESCRIPTION
**Pull request by DaveEslinger**
Should be working as V 1.0

### CODE REVIEW

File: QNSPECT/metadata.txt [hunk 1]
```diff
@@ -6,11 +6,12 @@
 name=QNSPECT
 qgisMinimumVersion=3.16
 description=QGIS Plugin for NOAA Nonpoint Source Pollution and Erosion Comparison Tool (NSPECT)
-version=0.0.1
+version=1.0
```
Comment (ar-siddiqui) [inline QNSPECT/metadata.txt:9]: This should be 0.0.2. This is just a small patch, shouldn't bump versions to 1.0, or else with this speed, we will soon be in two digits versions.

In fact, there should be a branching and versioning strategy devised for the long term.

File: QNSPECT/metadata.txt [hunk 2]
```diff
@@ -23,7 +24,7 @@ hasProcessingProvider=yes
 # changelog=
 
 # Tags are comma separated with spaces allowed
-tags=analysis, ecology, flood, flow, processing, erosion, pollution, environment, scenario, RUSLE, nonpoint, non-point, python, NSPECT, source, comparison, screening
+tags=analysis, ecology, flood, flow, processing, erosion, pollution, environment, scenario, RUSLE, nonpoint, non-point, python, QNSPECT, source, comparison, screening
 
 homepage=https://github.com/NOAA-OCM/QNSPECT
```
Comment (ar-siddiqui) [inline QNSPECT/metadata.txt]: This should be updated to the new home page.

File: QNSPECT/metadata.txt (hunk 1 above)
Comment (DaveEslinger) [reply QNSPECT/metadata.txt:9]: We (NOAA) use the protocol that 1.0 is the first complete version.  Minor changes will increase the additional digits, depending on their significance.

File: QNSPECT/metadata.txt (hunk 2 above)
Comment (DaveEslinger) [reply QNSPECT/metadata.txt]: Good catch, thanks.

**Pull request merged**